### PR TITLE
Change slot to opening in schema

### PR DESCRIPTION
**Pull request by khangnh107**
Commit to change "slot" to "opening" in database schema

### CODE REVIEW

**Review by linhkhanhhoang — APPROVED** [review]

File: server/models/OpeningBatch.js
Comment (hoangr1010) [inline server/models/OpeningBatch.js:1]: "open_list" should named "opening_list" for follow common naming

**Review by hoangr1010 — CHANGES_REQUESTED** [review]
Addition:
- Opening.js: request_id
- Request.js: opening_id

File: server/models/OpeningBatch.js
Comment (khangnh107) [reply server/models/OpeningBatch.js:1]: Should be fixed now

**Review by hoangr1010 — APPROVED** [review]
LGTM!

**Pull request merged**